main.js

```diff
@@ -1,71 +1,39 @@
 'use strict';
 
-// brings in the assert module for unit testing
-const assert = require('assert');
-// brings in the readline module to access the command line
-const readline = require('readline');
-// use the readline module to print out to the command line
-const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
-});
-
-
-const pigLatin = (word) => {
-
-  // Your code here
-
+  
```

Comment:
you are really close, and heading in the right direction but this logic is still a bit off.